test/Microsoft.AspNetCore.Routing.Tests/Constraints/StringRouteConstraintTest.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Routing.Constraints;
+using Moq;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Routing.Tests.Constraints
```

Comment:
should be the same as the product namespace `Microsoft.AspNetCore.Routing.Constraints`